src/templates/finops-hub/modules/hub.bicep

```
@@ -46,6 +46,9 @@ param remoteHubStorageKey string = ''
 @description('Optional. Enable telemetry to track anonymous module usage trends, monitor for bugs, and improve future releases.')
 param enableDefaultTelemetry bool = true
 
+@description('Optional. Enable infrastructure encryption on the storage account.')
+param enableInfrastructureEncryption bool = true
```

Update the default and description here if you end up changing the main.bicep version.

Small nit: Can you move this above the default telemetry parameter? I'd like to keep that last.

```suggestion
 @description('Optional. Enable infrastructure encryption on the storage account. Default = false.')
param enableInfrastructureEncryption bool = false
```